ClosedXML/Excel/Exceptions/EmptyTableException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace ClosedXML.Excel.Exceptions
+{
+    public class EmptyTableException : Exception
```

Comment:
What do you think about inheriting all ClosedXML exceptions from a single one (e.g. `abstract ClosedXmlException`)? This would make them easier to catch.

Reply:
Yes, definitely.